index.html

```diff
@@ -15,5 +15,69 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header>
+        <nav class="nav-bar">
+            <img src="/img/karma-logo.svg" height="50px" width="50px" alt="logo image of karma" class="logo_pic">
+            <a href="#" class="toggle-button">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </a>
+            <ul class="nav-link">
+                <li class="nav-item"><a href="#">Meet Karma</a></li>
+                <li class="nav-item"><a href="#">How It Works</a></li>
+                <li class="nav-item"><a href="#">Store</a></li>
+                <li class="nav-item"><a href="#">Blog</a></li>
+                <li class="nav-item"><a href="#">Help</a></li>
+                <li class="nav-item"><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <!--main section starts here-->
+    <main>
+        <section id="intro">
+            <h1 class="title">Introducing Karma</h1>
+            <p class="description">Bring WiFI with you, everywhere you go.</p>
+            <a class="btn" href="#service_card">Learn More</a>
+        </section>
+
+        <!--service section starts here-->
+        <section id="service-container">
+            <h2>Everyone needs a little Karma</h2>
+            <div class="cards">
+            <article class="service-card">
+                <img src="/img/icon-devices.svg" alt="desktop and mobile device pics">
+                <h3>Internet for all devices</h3>
+            </article>
+            <article class="service-card">
+                <img src="/img/icon-coffee.svg" alt="image of a coffee cup">
+                <h3>Boost your productivity</h3>
+            </article>
+            <article class="service-card">
+                <img src="/img/icon-refill.svg" alt="refilling petrol picture">
+                <h3>Pay as you go</h3>
+            </article>
+            </div>
+        </section>
+
+        <section id="get_karma" class="grid">
```

Comment:
Most of your class and id names are sensibly chosen - good work :) Here the class name of "grid" is quite generic, but there are other grids on the page which have quite different properties. In this case it would be better for the class name to be slightly more specific to the feature on the page.

Reply:
Hi Paul, thank you for your suggestion. I made the change to the class name to get_karma-grid. Kind regards.